pkgs/misc/emulators/pcsx2/default.nix

```diff
@@ -1,17 +1,19 @@
-{ alsaLib, cmake, fetchFromGitHub, gcc-unwrapped, gettext, glib, gtk3, harfbuzz
-, libaio, libpcap, libpng, libxml2, makeWrapper, perl, pkgconfig, portaudio
-, SDL2, soundtouch, stdenv, udev, wrapGAppsHook, wxGTK, zlib
+{ alsaLib, cmake, fetchFromGitHub, fmt, gcc-unwrapped, gettext, glib, gtk3
+, harfbuzz, libaio, libpcap, libpng, libpulseaudio, libsamplerate
+, libxml2, makeWrapper, perl, pkgconfig, portaudio, SDL2, soundtouch
+, stdenv, udev, wrapGAppsHook, wxGTK, zlib
 }:
 
 stdenv.mkDerivation {
   pname = "pcsx2";
-  version = "unstable-2020-10-10";
+  version = "unstable-2020-11-13";
 
   src = fetchFromGitHub {
     owner = "PCSX2";
     repo = "pcsx2";
-    rev = "7e2ccd64e8e6049b6059141e8767037463421c33";
-    sha256 = "0c7m74ch68p4y9xlld34a9r38kb2py6wlkg4vranc6dicxvi1b3r";
+    fetchSubmodules = true;
```

Comment:
This line can be removed now.

Reply:
pcsx2 already uses 2 submodules for other reasons so no, not really. Even if they're left unused for now they might be useful in the future.

Reply:
One is gtest, which we don't need since we aren't building any tests. The other is xz, which is already packaged in nixpkgs.
The line should be removed and xz added to `buildInputs`.
I already built and tested pcsx2 with those changes.